tests/tests.cpp

```diff
@@ -56,6 +61,27 @@ size_t checksize(size_t& n, size_t added)
     return n;
 }
 
+TEST(MegaApi, getMimeType)
+{
+
+    vector<unique_ptr<thread>> threads;
+
+    atomic<int> successCount = 0;
+
+    for (int i = 0; i < 20; ++i)
+    {
+        threads.emplace_back(unique_ptr<thread>(new thread([&successCount]() {
+            if (::mega::MegaApi::getMimeType("nosuch") == nullptr) ++successCount;
+            if (::mega::MegaApi::getMimeType("323") == string("text/h323")) ++successCount;
+            if (::mega::MegaApi::getMimeType("zip") == string("application/x-zip-compressed")) ++successCount;
+        })));
+    }
+
+    for (auto& a : threads) a->join();
```

Comment:
To prevent bugs, I'd prefer if we always used bodies no matter how long the statement

Reply:
Also, `a` deserves a better name ;)

Reply:
that's a lot of extra lines for very little code though, and it's only a unit test  - I could keep it compact with for_each and a lambda, but I think this formulation is much easier to read?  thanks

Renamed to 't' for thread, though its scope is only a single line so we don't need to remember its purpose for long :-)